src/fondue/Fondue.js

```diff
@@ -188,28 +188,33 @@ export default class Fondue {
 	// Usage:
 	//   fondue.features
 	get features() {
+		const featureResult = {};
 		const result = this._raw("GSUB");
 		const featuresRaw = result;
-		const features = result.featureList.featureRecords.map(
-			(record) => record.featureTag
-		);
-		const scripts = result.scriptList.scriptRecords.map(
-			(record) => record.scriptTag
-		);
-		// features = features.filter((feature, index) => features.indexOf(feature) === index); // remove doubles
-		const featureResult = {};
-		features.forEach((feature, index) => {
-			if (!featureResult[feature]) {
-				featureResult[feature] = {
-					...layoutFeature[feature],
-					scripts: {},
+
+		if (result) {
+			const features = result.featureList.featureRecords.map(
+				(record) => record.featureTag
+			);
+			const scripts = result.scriptList.scriptRecords.map(
+				(record) => record.scriptTag
+			);
+			// features = features.filter((feature, index) => features.indexOf(feature) === index); // remove doubles
```

Comment:
Can this code be removed?

Reply:
This was already present in the original code. I wanted to leave it in until I'm actively going to dig into this code, in case this is a clue. (For now I just put that `if` around it).